protoc-gen-go/grpc/grpc.go

```diff
@@ -223,6 +223,7 @@ func (g *grpc) generateService(file *generator.FileDescriptor, service *pb.Servi
 	}
 
 	// Service descriptor.
+	g.P("func Get", servName, "ServiceDescriptor() ([]byte, []int) { return ", file.VarName(), ", []int{", strconv.Itoa(index), "} }")
```

Comment:
Go style: is "Get" OK?

Also: people sometimes name their services something like "FooService"; This would result in `GetFooServiceServiceDescriptor`.  Messages have a `Descriptor` method.  Should this be `"func ", servName, "Descriptor()"`?

Reply:
I think the `Get` prefix is fine. In the case you mentioned where a proto `service` has a redundant name, I think it's okay for the generator to include the redundancy.